sql/policy_internal.sql

```diff
@@ -76,15 +76,26 @@ BEGIN
       AND (ch.status = 0 OR ch.status = 3)
   LOOP
     IF chunk_rec.status = 0 THEN
-       PERFORM @extschema@.compress_chunk( chunk_rec.oid );
+      BEGIN
+        PERFORM @extschema@.compress_chunk( chunk_rec.oid );
+        EXCEPTION WHEN OTHERS THEN
+        -- keep looping
```

Comment:
We shouldn't ignore the errors silently. Please use `RAISE WARNING` to log them to the server log.

Reply:
Thanks, fixed as suggested.

Reply:
Let's also add the original error message, to help the user figure what's going on.
I guess something like `RAISE ..... USING DETAIL = MESSAGE_NAME, ERRCODE = RETURNED_SQLSTATE` should work.

Some docs:
https://www.postgresql.org/docs/current/plpgsql-control-structures.html#PLPGSQL-EXCEPTION-DIAGNOSTICS
https://www.postgresql.org/docs/current/plpgsql-errors-and-messages.html